tests/index.js

```diff
@@ -0,0 +1,40 @@
+const tap = require('tap')
+const Fastify = require('fastify')
+const { Type } = require('@sinclair/typebox')
```

Comment:
Sorry, just for a bit of context. Previously Fastify would throw with a `FST_ERR_SCH_VALIDATION_BUILD` exception if it was not configured specifically for the `kind` and `modifier` TypeBox keywords (noting that this error was during the in Fastify schema compilation phase which happens following a call to `.listen(...)`). 

Because of the above requirement to configure AJV for TypeBox use (and as a convenience to users) the `ajvTypeBoxPlugin` was included in this package to help users with the necessary setup. However the most recent version of TypeBox mitigates the need such setup.

The tests you see below are merely to ensure Fastify no longer throws the `FST_ERR_SCH_VALIDATION_BUILD` for TypeBox types in absence of configuration.

As for testing the schemas themselves (or that data is received as expected on the route), that testing is probably best implemented in Fastify itself as TypeBox types are just JSON schema.

> Note: Additional runtime tests above and beyond what has been implemented here may signal that similar tests should also be implemented in the other provider package for `json-schema-to-ts`. (see https://github.com/fastify/fastify-type-provider-json-schema-to-ts).

An alternative test may have been to verify the schema compilation on `fast-json-stringify` directly, however testing against Fastify directly helps ensure an end to end test.

Reply:
got it, thank you for explanation!